backend/migrations/20230724125220_trigram.down.sql

```diff
@@ -0,0 +1,2 @@
+-- Add down migration script here
+DROP EXTENSION pg_trgm;
```

Comment:
Remove as there's no longer a need for? If we don't get it to run, we should remove it.

Reply:
?